src/constants/errors.ts

```diff
@@ -13,3 +14,25 @@ export type ErrorParams =
 export const DEFAULT_SOMETHING_WENT_WRONG_ERROR_PARAMS = {
   errorStringKey: STRING_KEYS.SOMETHING_WENT_WRONG,
 };
+
+/**
+ * Error codes provided by ethereum EIPS 1474 and 1193
+ * https://eips.ethereum.org/EIPS/eip-1474#error-codes
+ * https://eips.ethereum.org/EIPS/eip-1193#provider-errors
+ *
+ * There error codes are valid for EIPS compliant EVM wallets (including Metamask) and Phantom wallet.
+ */
+export enum EipErrorCodes {
```

Comment:
if people care, i'm also happy to split this into 2 enums since a wallet may choose to implement EIP-1193 and not EIP-1474, but I don't think we need to tbh.